src/RichText.js

```diff
@@ -0,0 +1,82 @@
+import {Editor, ContentState, convertFromHTML, DefaultDraftBlockRenderMap, DefaultDraftInlineStyle, CompositeDecorator} from 'draft-js';
+const ReactDOM = require('react-dom');
+var sdk = require('./index');
+
+const BLOCK_RENDER_MAP = DefaultDraftBlockRenderMap.set('unstyled', {
+    element: 'p' // draft uses <div> by default which we don't really like
```

Comment:
Good comment but you're missing the bit which links what you're saying to the code:
```javascript
// draft uses <div> by default which we don't really like
// draft uses <div> by default which we don't really like, so use <p> instead
```